### ISSUE TITLE
[Fenix] Onboarding colours have contrast issues

### ISSUE THREAD

**Issue by AmyYLee** [mozilla-mobile/fenix#5711]
Hi, 

Recommendations for fixing some contrast issues for the Onboarding theme. Please see below:

UPDATED Spec for colouring: Please go into "Inspect" mode to view colour hex. Let me know if you have questions @amilaneni99

https://share.goabstract.com/9507950f-cabb-415d-b4fa-1e9ee5229621?mode=design&sha=cf4d6e8baf5e4e3bb504ab793a2e94571e1b043d

**Comment by amilaneni99** [comment]
If this issue is open, I'd like to work on this. Thanks.

**Comment by vipulasri** [comment]
@NotWoods I would like to work on this.

**Comment by NotWoods** [comment]
Sorry @vipulasri, @amilaneni99 commented first so I'll let them take on this issue.

**Comment by AmyYLee** [comment]
@amilaneni99 Updated colour (new gradient) for the sign-in component

**Comment by vespertineSiren** [comment]
Please assign , I would like to work on this 👍

**Comment by vespertineSiren** [comment]
@AmyYLee  should I assume that since both the light and dark modes of the onboarding colors are the same, I should consolidate the background to one drawable resource or should I keep them separate for changes in the future?

**Comment by AmyYLee** [comment]
@vespertineSiren Hi, wanted to clarify which colours are you referring to that are the same? The gradient on the "Sign in to Firefox" card should be the same gradient for Light/Dark. The other colours are different.

Different for Light/Dark theme
<img width="451" alt="Screen Shot 2019-11-21 at 2 27 17 PM" src="https://user-images.githubusercontent.com/5992746/69370083-38129f80-0c6b-11ea-905c-49352f7f7176.png">

Same for Light/Dark Theme
<img width="848" alt="Screen Shot 2019-11-21 at 2 00 14 PM" src="https://user-images.githubusercontent.com/5992746/69370089-3e088080-0c6b-11ea-837b-47dfdc40b1e1.png">

**Comment by sv-sdeiac** [comment]
Hello, I investigate this issue and it seems that only the  "Sign in" card is changed to the new colors. The other cards are not changed. 

Devices: 
- Nokia 6 (Android 7.1.1);
- Motorola Moto G6 (Android 8.0);
- Xiaomi Mi 8 Lite (Android 9);
- Samsung Galaxy Note 8 (Android 9).

Till I have new information regarding this issue, I'll remove the qa:neede flag, thanks.

**Comment by sv-sdeiac** [comment]
The issue is still visible using the latest version of Nightly 1/10 #20100616.

Devices:
- Xiaomi Mi 8 Lite (Android 9);
- LG G7 fit (Android 8.1).

Since the issue is still reproducible, I'll remove the qa:needed flag till I have new updates, thanks.

**Comment by NotWoods** [comment]
Unassigned so someone else can pick up the remaining work if they want. Feel free to comment if you want to be reassigned :)

**Comment by mohammaduvez** [comment]
@NotWoods I want to pick up. Please assign

**Comment by sv-sdeiac** [comment]
Verified as fixed using the latest version of Firefox Preview Nightly 2/6 #20370604.

Devices: 
- Nokia 6 (Android 7.1.1);
- Samsung Galaxy S8 (Android 9);
- Google Pixel 3 (Android Q).
